msgpack/fallback.py

```diff
@@ -696,7 +696,7 @@ def _pack(self, obj, nest_limit=DEFAULT_RECURSE_LIMIT,
                 else:
                     raise PackValueError("Bytes is too large")
                 return self._buffer.write(obj)
-            if check(obj, (Unicode, bytes, memoryview)):
+            if check(obj, (Unicode, bytes)):
```

Comment:
I would like to keep memoryviews separate to raise errors with less confusing messages. E.g., I can then replicate the same errors raised in C and Python.
I can also eliminate some unnecessary branches that way.